model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/state_utils/states.py

```diff
@@ -5,7 +5,6 @@
 #
 # Please, refer to the LICENSE file in the root directory.
 # SPDX-License-Identifier: BSD-3-Clause
```

Comment:
This file has a bunch of `gtx.Field` that are missing the `gtx.Dims` (if the secondary objective of this PR is also to fix these things?)